src/utils.domNodeDisposal.js

```diff
@@ -85,7 +90,7 @@ ko.utils.domNodeDisposal = new (function () {
                 node.parentNode.removeChild(node);
         },
 
-        "cleanExternalData" : function (node) {
+        cleanExternalData : function (node) {
```

Comment:
Does this un-export `cleanExternalData`, and is that deliberate? I thought this was an intentional extensibility point.